src/components/form/form.scss

```diff
@@ -25,6 +25,10 @@
     }
 }
 
+limel-code-editor {
```

Comment:
Tested. Works :)

Not sure why we only want it on code editors nested in forms, but don't know the context 😅

Reply:
we have it for the `check-box` as well. 😉
In our case all input fields have a `helper-text`, so extra space is reserved for it, however, `limel-code-editor` doesn't have this feature, so the gap looks significantly smaller. Until now we were using this component only at the bottom of the form, so there was no need for adding a margin-bottom.